rewriter/Minitest.cc

```diff
@@ -154,12 +222,35 @@ unique_ptr<ast::Expression> runSingle(core::MutableContext ctx, ast::Send *send)
         return nullptr;
     }
 
+    if (send->fun == core::Names::testEach() && send->args.size() == 1 && send->block != nullptr) {
+        auto &expr = send->args.front();
+        if (!ast::isa_tree<ast::Array>(expr.get()) && !ast::isa_tree<ast::UnresolvedConstantLit>(expr.get())) {
+            if (auto e = ctx.state.beginError(expr->loc, core::errors::Rewriter::NonConstantTestEach)) {
+                e.setHeader("`{}` can only be used with constants or array literals", "test_each");
```

Comment:
Can you add something to the error message here suggesting how to refactor this to use a constant instead of a local? Something like `e.addErrorSection(ErrorSection(...))`.